schemas/ispyb/stored_programs/sp_retrieve_image_quality_indicators.sql

```diff
@@ -0,0 +1,24 @@
+DELIMITER ;;
+CREATE OR REPLACE DEFINER=`ispyb_root`@`%` PROCEDURE `retrieve_image_quality_indicators`(p_dataCollectionId int unsigned,
+p_imageNumber int unsigned)
+READS SQL DATA
+COMMENT 'Returns a single-row result set with the image quality indicators for a given data collection id and image number.'
+BEGIN
+	IF dataCollectionId IS NOT NULL THEN
+		IF imageNumber IS NOT NULL THEN
+
+			SELECT * FROM ImageQualityIndicators iqi
```

Comment:
`SELECT *` is generally discouraged in production code, let's specify the columns instead (even if that happens to list all of them ...)